src/ambianic/pipeline/ai/fall_detect.py

```diff
@@ -159,6 +180,11 @@ def fall_detect(self, image=None):
                     elif label == 'right shoulder' or label == 'right hip':
```

Comment:
@bhavikapanara I noticed this earlier, but forgot to mention. Since the order of points is important in relation to vector arithmetics, we should make sure that pose_vals_list[i][0] is always shoulder and pose_vals_list[i][1] is always hip. 
Since the or statement does not guarantee that, is it possible that we will add them to the list in reverse order [hip, shoulder] which will reverse the direction of the vector and the effective detection will be for someone standing up instead of falling. 

Instead of iterating over each keypoint items list and hoping they come always in the order we expect, should we optimize it to just use a lookup for 'left shoulder', 'left hip', 'right shoulder' and 'right hip'?

That would not only remove the possibility of iterating out of order but it will also remove the cycles iterating over keypoints we don't use.

Thoughts?

Reply:
@bhavikapanara thoughts ^^^?

Reply:
@ivelin I have made changes to tackle the problem of `"make sure that pose_vals_list[i][0] is always shoulder and pose_vals_list[i][1] is always hip."`